npm/webdriverio.json

```diff
@@ -0,0 +1,6 @@
+{
+  "versions": {
+    "4.2.16-s": "github:sfali23/typings-webdriverio/standalone#49ae26a9976863a61619b715e49ec2aeefa44257",
```

Comment:
They are not distinguished, but when you use `wdio` the API works in synchronously, I guess using `wdio-sync` package.